gradle/copy-rest-api-spec-to-global.gradle

```diff
@@ -0,0 +1,61 @@
+
+/**
+ * Copies all of the REST api specs (including core, modules, plugins, and x-pack) to common location for the REST YAML test.
+ * This plugin is expected to only be applied to applied to the root project.
+ */
+File rootBuildDir = rootProject.buildDir
```

Comment:
Ok, so I think we decided against centralizing _all_ specs because I came late to the party with the whole cache hit issue. Sorry about that. Given that then the only _common_ specs are those in `:rest-api-spec`, let's just add this logic to that project, exposing it's specs as a build artifact. Kinda what we already do ☹️